src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -179,56 +164,66 @@ internal static SqlConnection GetSqlConnection(ConnectionInfo connInfo)
             return null;
         }
 
-        internal void InitializeBackup(CDataContainer dataContainer, SqlConnection sqlConnection)
+        internal BackupConfigInfo GetBackupConfigInfo(CDataContainer dataContainer, SqlConnection sqlConnection, string databaseName)
         {
-            this.backupUtilities.Initialize(dataContainer, sqlConnection);
-        }
-
-        internal void SetBackupInput(BackupInfo input)
-        {   
-            this.backupUtilities.SetBackupInput(input);
+            BackupOperation backupOperation = new BackupOperation();
+            backupOperation.Initialize(dataContainer, sqlConnection);
+            return backupOperation.GetBackupConfigInfo(databaseName);
         }
 
-        internal void PerformBackup()
+        internal BackupOperation SetBackupInput(CDataContainer dataContainer, SqlConnection sqlConnection, BackupInfo input)
         {
-            this.backupUtilities.PerformBackup();
+            BackupOperation backupOperation = new BackupOperation();
+            backupOperation.Initialize(dataContainer, sqlConnection);
+            backupOperation.SetBackupInput(input);
+            return backupOperation;
         }
-
-        internal BackupConfigInfo GetBackupConfigInfo(string databaseName)
+                
+        internal void PerformBackup(BackupOperation backupOperation)
```

Comment:
do we need this method?  it looks like it's just directly calling another method.